README.md

```diff
@@ -51,6 +51,7 @@ This work is licensed under a [Creative Commons Attribution-ShareAlike 2.0 Gener
 - It’s easy for people to feel overwhelmed by the social interactions, sound level and business at conferences. A quiet room, which is a (somewhat) noise insulated room where there is no talking, can give these attendees a chance to decompress. It’s also a good place for people that need to focus on some work. All you need is a room with some chairs/desks/beanbags.
 Make sure that your quiet room is in a place that's easily and discretely accessible.
 - Similarly, you can also consider to have a separate prayer room.
+- You cannot have too many signs to guide people. Design them well ahead with a consistent color scheme and the conference logo. Invest in stands so they can be placed in exactly the right place, not Blu-Tacked to a wall. By the time you are setting up for the conference, you will know the venue layout well. Bring someone who has never seen the place before. Get them to start at the railway station or bus stop and navigate to the checkin desk. Repeat from the carpark. Repeat to get to speaking halls. Place a sign anywhere that it's not obvious where to go. As well as signs for checkin and speaking halls, consider signs for food, restrooms, and bus pickup points for off-venue events.
```

Comment:
I disagree. I have been to events which had signs all over the place, which made moving around very confusing: people were constantly checking the walls and reading signs to be sure that they don't miss anything. 

I'd suggest a more moderate wording here

Reply:
OK, I can see that's possible. I have reworded my PR.

My own experience in general has been too few rather than too many and I remember one conference that did a much better job than I typically find. I would like conference organisers to *consider* and *plan* for this. If they do they'll probably come up with a good answer.

I'm happy for you to adapt in any way you see fit.